src/main/java/org/cascadebot/cascadebot/utils/diff/Diff.java

```diff
@@ -0,0 +1,4 @@
+package org.cascadebot.cascadebot.utils.diff;
+
+public interface Diff {
```

Comment:
Instead of an empty interface, could have `getOld()` and `getNew()` functions?

Reply:
because the things this is being used for don't all have old a new, this is also being used now in our CollectionDiff stuffs which doesn't have old or new, this interface is here so I have a common thing i can use when I'm referring to either the collection diff, or the old/new diff.